languages/r-3/config.json

```diff
@@ -0,0 +1,7 @@
+{
+    "artifacts": [
+        {"source":"/home/algo/.local", "destination":"/home/algo/.local/"},
+        {"source":"/opt/algorithm", "destination":"/opt/algorithm/"},
+        {"source":"/home/algo/.Renviron", "destination":"/home/algo/.Renviron"}
```

Comment:
I don't think we need this. We should be instead copying over the default location that CRAN installs packages